test/react_test_utils_test.dart

```diff
@@ -102,16 +102,59 @@ void main() {
       group('dragStart', () => testEvent(Simulate.dragStart, 'dragStart'));
       group('drop', () => testEvent(Simulate.drop, 'drop'));
       group('focus', () => testEvent(Simulate.focus, 'focus'));
+      group('gotPointerCapture',
+          () => testEvent(Simulate.gotPointerCapture, 'gotPointerCapture'));
       group('input', () => testEvent(Simulate.input, 'input'));
       group('keyDown', () => testEvent(Simulate.keyDown, 'keyDown'));
       group('keyPress', () => testEvent(Simulate.keyPress, 'keyPress'));
       group('keyUp', () => testEvent(Simulate.keyUp, 'keyUp'));
+      group('lostPointerCapture',
+          () => testEvent(Simulate.lostPointerCapture, 'lostPointerCapture'));
       group('mouseDown', () => testEvent(Simulate.mouseDown, 'mouseDown'));
       group('mouseMove', () => testEvent(Simulate.mouseMove, 'mouseMove'));
       group('mouseOut', () => testEvent(Simulate.mouseOut, 'mouseOut'));
       group('mouseOver', () => testEvent(Simulate.mouseOver, 'mouseOver'));
       group('mouseUp', () => testEvent(Simulate.mouseUp, 'mouseUp'));
       group('paste', () => testEvent(Simulate.paste, 'paste'));
+      group(
+          'pointerCancel',
+          () => testEvent(
```

Comment:
The formatter on these got really gross output, id try manually formatting them again and re-running the formatter. :/

Reply:
FYI @kealjones-wk @joebingham-wk manual formatting has no effect on dartfmt since it ignores existing whitespace when formatting.